server/command_hooks.go

```diff
@@ -44,44 +57,45 @@ func (p *Plugin) emitStatusChange() {
 func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
 	configuration := p.getConfiguration()
 
-	if !strings.HasPrefix(args.Command, "/"+CommandTrigger) {
-		return &model.CommandResponse{
-			ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
-			Text:         fmt.Sprintf("Unknown command: " + args.Command),
-		}, nil
-	}
+	if strings.HasPrefix(args.Command, "/"+CommandTrigger) {
+
+		if strings.HasSuffix(args.Command, "true") {
```

Comment:
Maybe use a `switch`? 1/5

Reply:
Maybe combing this to a switch after `/ephemeral` is merged?  Both plugins use the if..else approach, and likely would be cleaner if the three were in a switch (and further commands)